mh/pat.py

```diff
@@ -2405,6 +2453,24 @@ def notify_layer_departure(self):
             ntc_data = pati.lp2_string(self.session.capcom_id)
             self.server.layer_broadcast(self.session, PatID4.NtcLayerOut,
                                         ntc_data, 0)
+        if self.session.local_info['circle_id'] is not None:
+            circle = self.session.get_circle()
```

Comment:
This line should be removed, since it's a leftover from old code.

Reply:
Done